### DISCUSSION TITLE
Change API output format

### DISCUSSION THREAD

**Discussion by wsligter** [FlowiseAI/Flowise#434] [Q&A]
I'm now only getting a string in the response from the flowise endpoint. I'd like to modify this to a JSON structure. I can't seem to find the part of the code where I can do this. Many thanks if anyone can point me to this!

**Comment by HenryHengZJ** [comment]
you can change it [here](https://github.com/FlowiseAI/Flowise/blob/main/packages/server/src/index.ts#L735)

**Comment by wsligter** [comment]
Many thanks!
________________________________
Van: Henry Heng ***@***.***>
Verzonden: donderdag 29 juni 2023 02:05
Aan: FlowiseAI/Flowise ***@***.***>
CC: wsligter ***@***.***>; Author ***@***.***>
Onderwerp: Re: [FlowiseAI/Flowise] Change API output format (Discussion #434)


you can change it here<https://github.com/FlowiseAI/Flowise/blob/main/packages/server/src/index.ts#L735>
